File: backend/alquiler/dominio/src/test/java/com/ceiba/alquiler/servicio/testdatabuilder/ClienteTestDataBuilder.java
package com.ceiba.alquiler.servicio.testdatabuilder;

import com.ceiba.alquiler.modelo.entidad.Cliente;
import com.ceiba.alquiler.modelo.entidad.ClienteId;

public class ClienteTestDataBuilder {
	private ClienteId id;
	private String identificacion;
	private String nombre;
	private String apellido;
	private String telefono;
	private String direccion;
	
	public ClienteTestDataBuilder() {
		id = new ClienteId(1L);
		identificacion = "1143354930";
		nombre = "Nombreprueba";
		apellido ="Apellidoprueba";
		telefono ="000000";
		direccion = "Carrera xx calle #xx-xx";
	}
	
	public ClienteTestDataBuilder conIdentificacion(String identificacion) {
		this.identificacion = identificacion;
		return this;
	}
	
	public Cliente build() {
		return new Cliente(id, identificacion, nombre, apellido, telefono, direccion);
	}
}

File: frontend/e2e/src/test/test-data-builder/cliente-testdatabuilder.ts
import { randomString } from "./utils/random-string";
export class ClienteTestDataBuilder {
    identificacion: string;
    nombre: string;
    apellido: string;
    telefono: string;
    direccion: string;
    constructor(){
        this.identificacion = randomString(8,'0123456789');
        this.nombre=  randomString(6,'0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ');
        this.apellido=  randomString(6,'0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ');
        this.telefono =  randomString(10,'0123456789');
        this.direccion=  randomString(16,'0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ');
    }
}